test/TestUtilities/Test.Utility/Signing/CertificateUtilities.cs

```diff
@@ -27,7 +27,7 @@ internal static AsymmetricCipherKeyPair CreateKeyPair(int strength = 2048)
 
         internal static string GenerateFingerprint(X509Certificate certificate)
         {
-            using (var hashAlgorithm = CryptoHashUtility.GetSha1HashProvider())
+            using (var hashAlgorithm = NuGet.Common.HashAlgorithmName.SHA256.GetHashProvider())
```

Comment:
NuGet is not a library for cryptography functions. Why are we exposing it, and why not call the BCL crypto class directly?

Reply:
Thanks for reviewing! 
I read from https://github.com/NuGet/NuGet.Client/pull/1929#discussion_r160293428 that, using NuGet.Common.HashAlgorithmName can have a better control on supported hash algorithms. Or else, we need to check in every call to see if the hash algorithm is supported. 
https://github.com/NuGet/Home/wiki/Package-Signatures-Technical-Details#SupportedAlgorithms

Reply:
As discussed, .NET doesn't provide an API to do the right thing regarding FIPS compliance on the TFMs/platforms we have to support, so we have to keep doing this :(